iphone/Maps/Core/Location/MWMLocationManager.mm

```diff
@@ -272,7 +263,7 @@ - (void)processLocationStatus:(MWMLocationStatus)locationStatus
   LOG(LINFO, ("Location status updated from", DebugPrint(self.lastLocationStatus), "to", DebugPrint(locationStatus)));
   self.lastLocationStatus = locationStatus;
   if (self.lastLocationStatus != MWMLocationStatusNoError)
-    self.frameworkUpdateMode |= MWMLocationFrameworkUpdateStatus;
+    GetFramework().OnLocationError((location::TLocationError)self.lastLocationStatus);
```

Comment:
1. Is it safe to casr MWMLocationStatusNoError to TLocationError? Maybe add a comment in the code that they should always be kept in sync?
2. Using locationStatus here instead of self.lastLocationStatus would be a bit faster.

Reply:
Let's proceed with the release and improve all nits later, thanks!